x/evm/keeper/querier.go

```diff
@@ -446,7 +446,10 @@ func QueryTokenAddressBySymbol(ctx sdk.Context, k types.ChainKeeper, n types.Nex
 		return nil, sdkerrors.Wrap(types.ErrEVM, fmt.Sprintf("token for symbol '%s' non-existent", symbol))
 	}
 
-	resp := types.QueryTokenAddressResponse{Address: token.GetAddress().Hex()}
+	resp := types.QueryTokenAddressResponse{
+		Address:   token.GetAddress().Hex(),
+		Confirmed: token.Is(types.Confirmed),
```

Comment:
Would it be more generic to just expose the status of the token than a `Confirmed` flag?

Reply:
yes, but:

1. We already error out if the token is non-existant, so we will never get that code in this field
2. the status type for the token is part of the `types` package rather than `exported` package, so I am trying to acoid exposing the type